src/core/math.d

```diff
@@ -177,38 +192,29 @@ unittest
  * Returns:
  *      f in precision of type `T`
  */
-@safe pure nothrow
 T toPrec(T:float)(float f) { pragma(inline, false); return f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:float)(double f) { pragma(inline, false); return cast(T) f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:float)(real f)  { pragma(inline, false); return cast(T) f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:double)(float f) { pragma(inline, false); return f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:double)(double f) { pragma(inline, false); return f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:double)(real f)  { pragma(inline, false); return cast(T) f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:real)(float f) { pragma(inline, false); return f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:real)(double f) { pragma(inline, false); return f; }
 /// ditto
-@safe pure nothrow
 T toPrec(T:real)(real f)  { pragma(inline, false); return f; }
 
 @safe unittest
 {
-    static float f = 1.1f;
-    static double d = 1.1;
-    static real r = 1.1L;
+    float f = 1.1f;
+    double d = 1.1;
+    real r = 1.1L;
```

Comment:
@ibuclaw was there any reason these need to be static? This makes the unittest import.

Reply:
To force the code generator (not CTFE) to do the work.  Practically the same is done in the test suite, so could remove if you really must.

Reply:
I'll remove it as part of #2871

Reply:
That PR seems to have [offended the gods of continuous integration](https://github.com/dlang/druntime/pull/2871#issuecomment-612826938), so I'd rather that be done here.

Reply:
I've added approxEqual, so the Win32 targets should now be happy with their inaccuracies.